modules/installing-ocp-agent-encrypt.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-encrypt_{context}"]
+= Optional: Encrypting the disk
+
+Use this procedure to encrypt your disk or partition while installing {product-title} with the Agent-based Installer.
+
+If you do not want to encrypt your disk or partition, proceed to the next section.
```

Comment:
I feel like you don't really need this sentence since you labeled the procedure itself as optional.